packages/framer-motion/src/animation/waapi/create-accelerated-animation.ts

```diff
@@ -0,0 +1,98 @@
+import { EasingDefinition } from "../../easing/types"
+import type { VisualElement } from "../../render/VisualElement"
+import type { MotionValue } from "../../value"
+import { animate } from "../legacy-popmotion"
+import { spring } from "../legacy-popmotion/spring"
+import { AnimationOptions } from "../types"
+import { animateStyle } from "./"
+
+const sampleDelta = 10 //ms
```

Comment:
Is there any science in picking this constant? Let's add a comment

Reply:
Good point - added

```
/**
 * 10ms is chosen here as it strikes a balance between smooth
 * results (more than one keyframe per frame at 60fps) and
 * keyframe quantity.
 */
```